src/pages/home/Home.js

```diff
@@ -0,0 +1,69 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import PostTile from "../../components/postTile/PostTile";
+import RedditHeader from "../../components/redditHeader/RedditHeader";
+import './Home.css';
+import redditLogo from "../../assets/redditlogo.png";
+import NoviFooter from "../../components/noviFooter/NoviFooter";
+
+function Home({error, toggleError, loading, toggleLoading}) {
+    const [redditPosts, setRedditPosts] = useState({});
+
+    useEffect(() => {
+        async function fetchData() {
+            toggleError(false);
+            toggleLoading(true);
+
+            try {
+                const {data} = await axios.get('https://www.reddit.com/hot.json?limit=15');
+                setRedditPosts(data.data.children);
+                toggleLoading(false);
+            } catch (e) {
+                console.error(e);
+                toggleError(true);
+                toggleLoading(false);
+            }
+
```

Comment:
Ik zie twee keer `toggleLoading(false)`. Als je deze in beide gevallen aanroept, kun je dat ook op het eind doen
```suggestion
            } catch (e) {
                console.error(e);
                toggleError(true);
            }
            
            toggleLoading(false);
```